Gruntfile.coffee

```diff
@@ -740,19 +726,18 @@ module.exports = (grunt) ->
 				tasks: [
 					"build"
 				]
-			lib_test:
-				files: "<%= jshint.lib_test.src %>"
-				tasks: [
-					"jshint:lib_test"
-				]
-			source:
-				files: "<%= jshint.lib_test.src %>"
-				tasks: [
-					"build"
+			js:
```

Comment:
Change is probably fine, but I don't recall if there was another reason for this split of tasks before

Reply:
The split of task was also about line ending wasn't it? Or maybe whitespace validation?

Reply:
Actually I think it was for a faster turnaround when someone was adding/improving tests. That way the whole JS didn't recompile when only working on testing changes. If no ones using it then it can go away though

Reply:
I'm not sure what it was being used for, but it wasn't for line endings (that's covered by .eslintrc.ci.json - which only Travis-CI is configured to use via wet-boew/wet-boew#7948).

In the wet-boew/wet-boew repo, it looks like it was originally introduced in wet-boew/wet-boew@fea3ff52c3721d273d4b9e172110565e7dd84eb0 (very first v4.0 commit) and eventually removed in wet-boew/wet-boew#6514. I guess the removals hadn't previously been ported to the theme repos.

Reply:
Btw I could potentially revise this to make the watch task auto-run eslint when JS files get modified, but I don't think it'd be worthwhile to do as a part of this PR since wet-boew/wet-boew doesn't currently do that. So I don't want to inadvertently put the themes ahead of it in that respect.